code/modules/clothing/gloves/color.dm

```diff
@@ -2,7 +2,7 @@
 	dying_key = DYE_REGISTRY_GLOVES
 
 /obj/item/clothing/gloves/color/yellow
-	desc = "These gloves provide protection against electric shock. The thickness of the rubber makes your fingers seem bigger."
+	desc = "These gloves provide protection against electric shock."
```

Comment:
Don't forget to undo the chunky fingers stuff for the gloves and update your PR title and body.